tests/sentry/snuba/metrics/test_query_builder.py

```diff
@@ -1013,6 +1014,101 @@ def test_translate_meta_results_with_duplicates():
     )
 
 
+@pytest.mark.parametrize(
```

Comment:
@iambriccardo JFYI, how we handle tests, instead of loops inside. I saw in you PR you do some loops to cover all cases.